src/main/java/com/conveyal/datatools/manager/utils/sql/SqlSchemaUpdater.java

```diff
@@ -25,31 +26,38 @@
 import static com.mongodb.client.model.Filters.not;
 
 /**
- * This class handles operations to detect whether postgresql tables
- * are consistent with the gtfs-lib implementation from the project dependencies,
+ * This class handles operations to detect whether postgresql tables are consistent with the gtfs-lib implementation,
  * and fills out missing tables/columns and changes column types if needed.
  */
 public class SqlSchemaUpdater implements AutoCloseable {
-    /** Maintain a list of scanned namespaces to avoid duplicate work. */
-    private Map<String, NamespaceCheck> checkedNamespaces = new HashMap<>();
+    public static final String VERSIONS = "VERSIONS";
+    public static final String EDITOR = "EDITOR";
+    public static final String SNAPSHOTS = "SNAPSHOTS";
+    public static final List<String> NAMESPACE_TYPES = Lists.newArrayList(VERSIONS, EDITOR, SNAPSHOTS);
 
+    /** Maintain a list of scanned namespaces to avoid duplicate work. */
+    private final Map<String, NamespaceCheck> checkedNamespaces = new HashMap<>();
     private final Connection connection;
+    /** Check that all tables for the namespace are present. */
     private final PreparedStatement selectNamespaceTablesStatement;
+    /** Check that columns for a given table are present. */
     private final PreparedStatement selectColumnStatement;
+    /** Check whether a namespace was deleted. */
+    private final PreparedStatement selectFeedStatement;
```

Comment:
Add `selectFeedStatement.close();` to the close method at the bottom of the class.

Reply:
Thanks, updated in https://github.com/ibi-group/datatools-server/pull/459/commits/ce776650df0f8e7ce2e81ef91c0db6fd82a48a54.